lib/util.js

```diff
@@ -232,7 +232,9 @@ const util = {
     fileMap.add([path.join(braveAppDir, 'theme', 'default_200_percent', 'brave'), path.join(chromeAppDir, 'theme', 'default_200_percent', 'chromium')])
     fileMap.add([path.join(braveAppDir, 'theme', 'default_100_percent', 'common'), path.join(chromeAppDir, 'theme', 'default_100_percent', 'common')])
     fileMap.add([path.join(braveAppDir, 'theme', 'default_200_percent', 'common'), path.join(chromeAppDir, 'theme', 'default_200_percent', 'common')])
+    fileMap.add([path.join(braveComponentsDir, 'resources', 'default_100_percent'), path.join(chromeComponentsDir, 'resources', 'default_100_percent')])
```

Comment:
is this a recursive copy? Because if so we don't need the one below it

Reply:
It is recursive but below is needed because it is copied to different folder. Newly copied Brave folder is just not used. I think it is fine.

Reply:
ah, right, because it will end up in `brave` instead of `chromium`, but then what's the point of naming the folder `brave` in the first place? Shouldn't we just name it 'chromium' and copy the whole thing together?

Reply:
then we could just have one copy for `braveComponentsDir, 'resources'` and not a dozen different entries

Reply:
^^ assuming we do the same thing with theme, etc.. we can just copy braveComponentsDir

Reply:
seems like we're making this way more complicated than it needs to be by using `brave`

Reply:
f/u issue - https://github.com/brave/brave-browser/issues/10080